### DISCUSSION TITLE
Trigger save row or automation on page load

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#9394] [Help] [answered]
Hello,
Is it possible to trigger a save row action, or an automation, when a user opens a certain page ?
My use case is to mark as "read" (by an sql update) the row entered by the user, without requesting him to click somewhere.

**Comment by avivkri** [comment, marked as answer]
Hello @NicolasFabre81,
On screen load actions could be of help.
![Budibase on screen load actions](https://user-images.githubusercontent.com/13772138/213696193-3b3fd250-c6ef-474f-8b69-d6ac36b08d8b.png)

**Reply by NicolasFabre81** [reply]
Never seen it thank you !

**Reply by NicolasFabre81** [reply]
But unfortunately we cannot access the screen's data so it is very limited

**Reply by avivkri** [reply]
I see. Yeah, App state may be of help too, if expected data is not directly available through the bindings offered.
